.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           command: docker run --rm --tty --interactive application:$CIRCLE_SHA1 python --version | grep "2.7"
       - run:
           name: Validate that image runs on Europe/London timezone
-          command: docker run --rm --tty --interactive application:$CIRCLE_SHA1 stat --format=%N /etc/localtime | grep "Europe/London"
+          command: docker run --rm --tty --interactive application:$CIRCLE_SHA1 date | egrep " (GMT|BST) "
```

Comment:
Not sure if it improves the test in anyway but is it worth outputting the timezone only, e.g.:
```
date +%Z
```

Reply:
Makes it clearer I think. Thank you, done!

Reply:
Well, I pushed the update but it seems the UI has a hard time catching up. Probably due to the current GitHub status:

> We continue to investigate issues with background jobs. Users may notice delayed updates, notifications, and Pages builds.